task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -48,8 +49,7 @@ spec:
         SKIP_MSG="We found 0 supported files"
         grep -q "$SKIP_MSG" stdout.txt || test_not_skipped=$?
 
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
-          '{result: "ERROR", timestamp: $date}')
+        HACBS_ERROR_OUTPUT=$(make_result_json -r ERROR)
```

Comment:
```suggestion
        HACBS_ERROR_OUTPUT=$(make_result_json -r ERROR -f 1)
```

Reply:
Or, sorry, we don't need a number for `ERROR` result, please ignore this comment.